client/src/components/application/Examine/ActionButtons.vue

```diff
@@ -184,12 +184,8 @@ export default {
       // if there is no expiry date, this NR is not approved-expired
       if (this.$store.getters.expiryDate == null) return false;
 
-      let expired_date = moment(this.$store.state.expiryDate, 'YYYY-MM-DD').clone()
-      let today = new moment()
+      if (this.$store.getters.currentState === 'EXPIRED') return true
```

Comment:
Clean!

Reply:
Is there a current state or nameRequest state enum?

Reply:
If not, it might be clean to implement one to minimize the strings in script. Your call tho!

Reply:
that might lead into quite a lot of work for him 😛 , but I agree it would be nice

Reply:
Absolutely, don't need to go too crazy. If there is some low hanging fruit giver, but no blocker from me :)

Reply:
I thought of adding an enum for state but as kial mentioned here its quite a lot of work. So I just followed existing structure